src/main/resources/assets/rockbottom/loc/fr_français.json

```diff
@@ -0,0 +1,182 @@
+{
+    "rockbottom": {
+        "loc.fr_francais": "Français - France",
```

Comment:
This should be the country code in capitals, followed by a space, followed by the name of the language in the language itself. For instance, US english has "US English" here.